tests/test-definition.yaml

```diff
@@ -3,12 +3,12 @@ dimensions:
   - name: opa
     values:
       # To use a custom image, add a comma and the full name after the product version
-      - 0.67.1,docker.stackable.tech/stackable/opa:0.67.1-stackable0.0.0-dev
```

Comment:
Can you please leave the custom image as comment, similar to how we do it in other operators?

https://github.com/stackabletech/trino-operator/blob/92feb2705b8cab6ceb351cd0b982dd81715a246d/tests/test-definition.yaml#L14-L19

Reply:
I can. Just didn't worked for me ( might be an MacOS thingy however )

Reply:
Thanks! Yeah I guess it's a MacOS special ^^

Reply:
Granted: https://github.com/stackabletech/opa-operator/pull/677/commits/47408ecc02206db0d272a092c26688427bf6f0ec